README.md

```diff
@@ -98,10 +98,10 @@ juju integrate mattermost-k8s postgresql-k8s:db
 Watch your deployment come to life:
 
 ```
-juju status --watch 1s
+watch -n 1 -c juju status --color
 ```
 
-(Press `Ctrl-C` to quit. Drop the `--watch 1s` flag to get the status statically. Use the `--relations` flag to view more information about your integrations.)
+(Press `Ctrl-C` to quit. Use the `juju status` to get the status statically. Use the `--relations` flag to view more information about your integrations.)
```

Comment:
We don't need `Press `Ctrl-C` to quit. Use the `juju status` to get the status statically.`
It can be deleted. That whole sentence is only relevant when `--watch` is used.

Also, we are calling them "relations" right? So the text is wrong. But the option is `--relations` is correct.

And we should mention `--storage` if we talk about `--relations`.

Reply:
Thx, I'll update the README